### ISSUE TITLE
Resume archive from existing archive

### ISSUE THREAD

**Issue by arthurlutz** [russelldavies/matrix-archive#3]
Follow up of #2 

When launching the command on a room that has already been archived, it would be nice to start off from where it stopped. Right now it resets the .yaml file and starts at the beginning. Could the timestamp in the last yaml message be used to resume from that message ?

**Comment by russelldavies** [comment]
So your case is where you have already made a full archive and then sometime later want to append to that existing archive?

**Comment by arthurlutz** [comment]
Yes. That would be awesome. (the current room I'm archiving has 8.8GiB of media)

**Comment by chrysn** [comment]
As for the media, it may be an option to just verify that a media item is present already and not re-download it (or only downloading it if it changed ... *can* media items change at all?).

An incremental mode for history would still make sense, but seeing #18 it may be a bit harder, and just skipping the 8GB of already-obtained data might be a good start already.
